tests/CMakeLists.txt

```diff
@@ -97,6 +97,17 @@ if (ENABLE_GTEST)
                             NUM_MPI_TASKS 4)
     endif()
 
+    if(SERAC_USE_PETSC)
+        blt_add_executable( NAME        petsc_smoketest
+                            SOURCES     petsc_smoketest.cpp
+                            OUTPUT_DIR  ${TEST_OUTPUT_DIRECTORY}
+                            DEPENDS_ON  PETSc mpi gtest
+                            FOLDER      serac/tests )
+        blt_add_test(NAME          petsc_smoketest
+                    COMMAND       petsc_smoketest
+                    NUM_MPI_TASKS 1 )
```

Comment:
It's probably a good idea to do some parallel thing with the smoke test since that is a possible major source of errors.

Reply:
I went ahead and just used one of PETSc's introductory examples